versioned-docs.md

```diff
@@ -0,0 +1,23 @@
+# Versioned Documentation
```

Comment:
Please link this somewhere as well https://github.com/fairlearn/fairlearn/issues/482

Reply:
This mention links it :-)

Reply:
Once checking in it won't